manifests/pool.pp

```diff
@@ -26,5 +27,10 @@
   concat::fragment { "dhcp_pool_${name}":
     target  => "${dhcp_dir}/dhcpd.pools",
     content => template('dhcp/dhcpd.pool.erb'),
+    order   => "10 ${sharednetwork}-10",
```

Comment:
Is there a specific reason order was added here?

Reply:
The reason is to be able to wrap all pool/pool6-definitions into their assigned-sharednetwork-segments (which requires a separate header and footer).

I start the order with `10 ` to not break any existing orders (since `10` ist default) and add the internal order as suffix (`-09` for the shared-network-header, `-10` for the pool/pool6-definitions, and `-11` for the shared-network-footer).